src/api_manager/config_manager_test.cc

```diff
@@ -283,6 +283,68 @@ TEST_F(ConfigManagerServiceNameConfigIdTest,
   ASSERT_EQ(0, sequence);
 }
 
+TEST_F(ConfigManagerServiceNameConfigIdTest, ResponseRolloutID) {
+  int sequence = 0;
+  std::shared_ptr<ConfigManager> config_manager(new ConfigManager(
+      global_context_,
+      [this, &sequence](const utils::Status& status,
+                        const std::vector<std::pair<std::string, int>>& list) {
+
+        ASSERT_EQ(1, list.size());
+        ASSERT_EQ(kServiceConfig1, list[0].first);
```

Comment:
nit: EXPECT_EQ here and other places where it would be appropriate.

Reply:
Done.